localenv/mock-account-servicing-entity/Dockerfile

```diff
@@ -59,4 +59,5 @@ COPY --from=builder /home/rafiki/localenv/mock-account-servicing-entity/build ./
 COPY --from=builder /home/rafiki/localenv/mock-account-servicing-entity/public ./localenv/mock-account-servicing-entity/public
 
 WORKDIR /home/rafiki/localenv/mock-account-servicing-entity
-CMD ["sh", "./node_modules/.bin/remix-serve", "build"]
\ No newline at end of file
+
+CMD ["sh", "-c", "NODE_ENV=production ./node_modules/.bin/remix-serve ./build/index.js"]
```

Comment:
(this comment also applies to [this same change here](https://github.com/interledger/rafiki/pull/2349/files#diff-3e0c81413453a46c12208866c9fb0686af8eaee0c49cfd701c1fdabd7aa3a660R60))

Couple changes here.

Target was changed to `./build/index.js` to resolve [this error](https://github.com/remix-run/remix/discussions/7586) which is pretty straightforward.

`NODE_ENV` changed to production to resolve [this error](https://github.com/remix-run/remix/discussions/7338). I don't fully understand what caused this to break nor why this was originally running with `NODE_ENV=development` (coming from docker-compose). I would think it should be production given it's running code built in production mode (confirmed here and in main by observing logs from `pnpm --filter mock-account-servicing-entity build`). Docker container's start fine and was able to use MASE UI's and frontends without issue.

Reply:
NODE_ENV=development is needed for things like the message server (`/packages/frontend/app/lib/message.server.ts`) to work over HTTP instead of HTTPS
```
export const messageStorage = createCookieSessionStorage<
  SessionData,
  MessageStorageFlashData
>({
  cookie: {
    name: '__message',
    path: '/',
    httpOnly: true,
    sameSite: 'lax',
    maxAge: ONE_MINUTE_IN_S,
    secrets: ['MY_SUPER_SECRET_TOKEN'],
    secure: process.env.NODE_ENV === 'production'
  }
})
```

If the environment is set to production and our local setup doesn't use https then this is gonna break.

Reply:
Looking into this... we haven't been able to reproduce such an issue yet. Seems like the check was added here (https://github.com/interledger/rafiki/pull/1459) so maybe only a problem with physical mobile device?

Also trying to get this working with `NODE_ENV=development`. I get this error which suggests we need to set a `REMIX_DEV_HTTP_ORIGIN` env var but cant find a value that its happy with:

```
/Users/blaircurrey/code/interledger/rafiki/node_modules/.pnpm/@remix-run+server-runtime@2.5.1_typescript@4.9.5/node_modules/@remix-run/server-runtime/dist/dev.js:17
  if (!origin) throw Error("Dev server origin not set");
                     ^
Error: Dev server origin not set
    at Object.broadcastDevReady (/Users/blaircurrey/code/interledger/rafiki/node_modules/.pnpm/@remix-run+server-runtime@2.5.1_typescript@4.9.5/node_modules/@remix-run/server-runtime/dist/dev.js:17:22)
    at Server.onListen (/Users/blaircurrey/code/interledger/rafiki/node_modules/.pnpm/@remix-run+serve@2.5.1_typescript@4.9.5/node_modules/@remix-run/serve/dist/cli.js:123:17)
    at Object.onceWrapper (node:events:627:28)
    at Server.emit (node:events:513:28)
    at emitListeningNT (node:net:1774:10)
    at processTicksAndRejections (node:internal/process/task_queues:81:21)
```

Reply:
After resolving a mobile bug I was finally able to test the toast notifications on mobile devices and they are indeed broken because `secure` is being set to `true`.

After a few renewed attempts to figure out how to configure remix-serve for development mode I found this comment by the remix staff engineer that suggests remix-serve isnt really intended to run in development mode (which makes sense, thats what `remix dev` is for) https://github.com/remix-run/remix/issues/8456#issuecomment-1885166316

I don't see a reason to use development mode here, given that we're not using HMR and our dockerfile is setup to install remix with prod dependencies. It is controlling this `secure` value but that's incidental - I think instead of trying to do `remix-serve` in dev mode we can just pass in a different environment variable to control the secure cookie value.

Reply:
added an insecure cookie flag here instead of running in dev mode: https://github.com/interledger/rafiki/pull/2349/commits/2db091cc06f3773a457a0709a7ea533b69cdd3d3